scripts/setup-seafile-database.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
```

Comment:
Since only a small fraction of users would use seafile with postgres, I think we'd better use separate scripts for mysql and postgres, instead of adding this wrapper script for both, so that we would have:

- setup-seafile.sh: setup seafile with sqlite3
- setup-seafile-mysql.sh: setup seafile with mysql
- setup-seafile-postgres.sh: setup seafile with postgres